cloud-data-protection/src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import App from './App';
+import './index.css';
+
+import {Provider} from 'react-redux';
+import store from './stores/Store';
+
+ReactDOM.render(
+  <Provider store={store}>
+      <BrowserRouter>
+          <App />
+      </BrowserRouter>
+  </Provider>,
+  document.getElementById('root')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
  document.getElementById("root")
```